templates/admin-server.template.yaml

```diff
@@ -359,11 +359,11 @@ Resources:
           - Action:
               - 'ec2:AssociateAddress'
               - 'ec2:DescribeAddresses'
-            Resource: '*'
+            Resource: !Sub "arn:${AWS::Partition}:*"
```

Comment:
@BradBloomingdale
hmm, let me check what exact resource we do need here.
But how the linter passed previously?

Reply:
Assuming it's an address, it would probably be something like this: 

"arn:${AWS::Partition}:ec2:${AWS::Region}:${AWS::AccountId}:network-interface/*"

Although thinking about it, these API calls are read-only, I'm not sure if it matters that much.

Reply:
> Assuming it's an address, it would probably be something like this:
> 
> "arn:${AWS::Partition}:ec2:${AWS::Region}:${AWS::AccountId}:network-interface/*"
> 
> Although thinking about it, these API calls are read-only, I'm not sure if it matters that much.
> 
> For the purposes of this template, I'd say a wildcard is fine.

Here - `For the purposes of this template, I'd say a wildcard is fine.` u meant to leave it as it was before, whether use the wildcard u suggested in quoted above?

Reply:
Rolled back prev wildcard

Reply:
@BradBloomingdale but for now linter will fail, need to bypass it somehow